src/main/java/com/commercetools/sync/services/impl/CustomerServiceImpl.java

```diff
@@ -26,10 +26,10 @@
 import static org.apache.commons.lang3.StringUtils.isBlank;
 
 
-public final class CustomerServiceImpl extends BaseServiceWithKey<CustomerDraft, Customer, CustomerSyncOptions,
+public final class CustomerServiceImpl extends BaseServiceWithKey<CustomerDraft, Customer, BaseSyncOptions,
```

Comment:
Why not using instead of changing the type.
```` java
new CustomerServiceImpl(CustomerSyncOptionsBuilder.of(shoppingListSyncOptions.getCtpClient()).build())
````

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/615/commits/a11ee22a5bf047fe480a0f1a1ddfc803eb4b9327